.github/labeler.yml

```diff
@@ -0,0 +1,10 @@
+# changes to documentation
+"area/documentation": doc/**/*
```

Comment:
The `doc` subdir in cobra is *not* for documentation. It's the documentation generation codebase of the cobra lib. My proposal in #1245 is to use `site` for the actual documentation of cobra.

Reply:
Updated this to reflect the docs generation: `doc-gen`